internal/exec/stages/files/filesystemEntries.go

```diff
@@ -512,3 +512,47 @@ func (s *stage) createEntries(entries []filesystemEntry) error {
 	}
 	return nil
 }
+
+// save the cex Volume keys
+func (s *stage) createCexVolumeKeys(config types.Config) error {
+	if len(config.Storage.Luks) == 0 {
+		return nil
+	}
+	s.Logger.PushPrefix("createCexVolumeKeys")
+	defer s.Logger.PopPrefix()
+	for _, luks := range config.Storage.Luks {
+		if luks.Cex.IsPresent() {
```

Comment:
Minor: instead of intending here, I'd reflow as:

```go
if !luks.Cex.IsPresent() {
    continue
}
...
```

Reply:
Will realign the flow.

Reply:
Done